crates/module-system/sov-modules-api/Cargo.toml

```diff
@@ -64,6 +64,7 @@ utoipa = { workspace = true, optional = true }
 utoipa-swagger-ui = { workspace = true, optional = true }
 unwrap-infallible = { workspace = true }
 nearly-linear = { workspace = true }
+arrayvec = { workspace = true, features = ["serde", "borsh"] }
```

Comment:
What do we use `arrayvec` for in the new PR? A quick grep didn't show any usage...

Reply:
It's a leftover. I started by replacing a String with a stack-String but later realized that we only use it in a macro that accepts 'static str. Will remove